jdl/converters/parsed-jdl-to-jdl-object/option-converter.spec.ts

```diff
@@ -91,12 +85,12 @@ describe('jdl - OptionConverter', () => {
         before(() => {
           convertedOptions = convertOptions({}, [
             {
-              optionValues: [MAPSTRUCT, COUCHBASE],
+              optionValues: ['mapstruct', 'elasticsearch'],
```

Comment:
More consistency is more valuable than decoupling. Jdl should be split into generator so each part create the dsl. Jdl should not be decoupled from generator but instead integrated into each of them

Reply:
Imho, this pr comes with regressions :  using constants helps folding to provide complete feature enhancement.  This pr breaks that useful feature.
I do not see any added value decoupling jdl and generator as the main end user usage is the jdl triggering generator :  jdl should be modularized and fragmented into generators instead of being decoupled and only covering partial coverage